frontends/web/src/routes/account/send/send.tsx

```diff
@@ -174,11 +174,18 @@ class Send extends Component<Props, State> {
         this.registerEvents();
         const account = this.getAccount();
         if (account && !account.coinCode.startsWith('eth-erc20-') && account.coinCode !== 'eth') {
-            this.qrCodeReader
-                .getVideoInputDevices()
-                .then(videoInputDevices => {
-                    this.setState({ hasCamera: videoInputDevices.length > 0 });
-                });
+            import('../../../components/qrcode/qrreader')
```

Comment:
I probably don't undestand something. I see that components/qrcode/qrreader imports `BrowserQRCodeReader` but this same type is already imported at the top of this file with `import { BrowserQRCodeReader } from '@zxing/library'`.

Doesn't that mean the `BrowserQRCodeReader` is already imported since the beginning when switching to send view? Then I guess I don't understand why this import here is needed.

Reply:
the BrowserQRCodeReader is already imported, is only for Typescript and only uses the type interface.

Reply:
Oh I see. So, before this change the qr reader was loaded due to this line?

```js
    private qrCodeReader: BrowserQRCodeReader = new BrowserQRCodeReader();
```

Reply:
correct

Reply:
:+1: